### PR TITLE
docs: update todolist doc

### PR DESCRIPTION
**Pull request by nabdelgadir**
Connect to #1908 and #1828. 
Minor fixes in the documentation for the TodoList example, including adding `lb4 repository` command, and other miscellaneous updates.

## Checklist

- [x] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [x] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [x] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

### CODE REVIEW

File: docs/site/todo-list-tutorial-controller.md [hunk 1]
```diff
@@ -221,6 +221,102 @@ export class TodoListController {
 }
 ```
 
+And the completed TodoListTodo controller should look like follows:
```
Comment (bajtos) [inline docs/site/todo-list-tutorial-controller.md]: Is it a good idea to keep a copy of the generated code in our docs? It means that whenever we change the CLI templates, we also need to update the copies in our docs. In my experience so far, we usually forget to do that and as a result, our documentation often contains outdated code.

Can we point users to the source code file on GitHub instead? For example:

```markdown
Check out our todo-list example to see the full source code generated for TodoListTodo controller:
[src/controllers/todo-list-todo.controller.ts](https://github.com/strongloop/loopback-next/blob/master/examples/todo-list/src/controllers/todo-list-todo.controller.ts)
```

File: docs/site/todo-list-tutorial-model.md [hunk 2]
```diff
@@ -68,10 +68,15 @@ Model TodoList was created in src/models/
...
+model. Add the following import statement and property to the `TodoList` model:
 
 #### src/models/todo-list.model.ts
 
+```ts
+import {hasMany} from '@loopback/repository';
+import {Todo} from './todo.model';
+```
+
 ```ts
```
Comment (bajtos) [inline docs/site/todo-list-tutorial-model.md]: Please remove the lines 78 and 80 to merge the new code block with the old code block into a single block.

File: docs/site/todo-list-tutorial-repository.md [hunk 3]
```diff
@@ -20,15 +20,22 @@ building a constrained version of `TodoRepository`.
 
 ### Create your repository
 
-In the `src/repositories` directory:
-
-- create `todo-list.repository.ts`
-- update `index.ts` to export the newly created repository
+From inside the project folder, run the `lb4 repository` command to create a
+repository for the TodoList model using the `db` datasource. The `db` datasource
```
Comment (b-admike) [inline docs/site/todo-list-tutorial-repository.md]: nitpick: TodoList -> `TodoList`

File: docs/site/Using-components.md [hunk 4]
```diff
@@ -15,6 +15,11 @@ allow easier extensibility of your Application.
 A typical LoopBack component is an [npm](https://www.npmjs.com) package
 exporting a Component class which can be added to your application.
 
+```sh
```
Comment (b-admike) [inline docs/site/Using-components.md]: I suggest we add a sentence or two on why we are installing the dependencies.

**Pull request merged**